buildconfig/pygame-stubs/rect.pyi

```diff
@@ -181,6 +181,16 @@ class Rect(object):
     def fit(self, left: float, top: float, width: float, height: float) -> Rect: ...
     def normalize(self) -> None: ...
     @overload
+    def __contains__(self, rect: Union[_RectStyle, Rect]) -> bool: ...
```

Comment:
hmm since an int is allowed for a contains check, we can probably do `Union[_RectStyle, Rect, int]` here

Reply:
Done!